### ISSUE TITLE
CI "Implementations" tests not helpful -- request they be removed

### ISSUE THREAD

**Issue by swcurran** [hyperledger/aries-rfcs#633]
@dhh1128 @TelegramSam -- suggest that we remove these tests in the CI.  I'd like to keep the "check links" test -- that's very useful.  But the Implementations tests are not obvious and the output of the "fail" messages are extremely difficult to understand.  I can't imagine a newbie would have a clue what to do.

In particular -- the "implementations" link is assumed to be unique per RFC, pointing at the code of the implementation, presumably.  Or, I guess, to evidence of a successful test.  People instead are just putting in a link to the repo of the implementation (e.g. the whole framework).  So far, the "easist" thing to do is just remove the implementation from the table.  Not what we want.

I'm not sure of a better way, but for now I'd like to just drop the feature.

Thoughts?

**Comment by dhh1128** [comment]
Yeah, if we don't believe in tracking impls in RFCs, we should turn this test off.

**Comment by dhh1128** [comment]
We might also want to turn off the test that checks for test anomalies, since we've been pushing to abandon the test suite as a community strategy.
